SignalMessaging/attachments/MediaMessageView.swift

```diff
@@ -456,6 +479,14 @@ class MediaMessageView: UIView, OWSAudioAttachmentPlayerDelegate {
         videoPlayer?.stop()
         videoPlayer?.view.removeFromSuperview()
         videoPlayer = nil
-        ViewControllerUtils.setAudioIgnoresHardwareMuteSwitch(false)
+        OWSAudioAttachmentPlayer.setAudioIgnoresHardwareMuteSwitch(false)
+    }
+}
+
+extension UIImage {
+    public convenience init?(named: String, inBundleForClass someClass: AnyClass) {
+        //    let image = UIImage(named: imageName, inBundleForClass: type(of:self))
```

Comment:
Can we remove this line?

Reply:
Done. Actually, since it seems like you're OK with this approach, I've moved this into `UIImage+OWS.h/m`